src/service/auth/auth.js

```diff
@@ -16,6 +17,18 @@ const CODE_TO_STATUS = {
   403: AUTH_STATUS_CODE.NEED_PERMISSION,
 };
 
+/**
+ * @param {Promise} promise
+ * @return {Promise<any>}
+ */
+const withSuccess = promise => {
```

Comment:
Why? Without success we will have errors too. Or we should includes that behavior on the requester level instead of standalone function, imo

Reply:
it's just a wrapper for throw error, will removed after implement `request` as is in `vault`

Reply:
:ok_hand: